README.md

```diff
@@ -249,6 +231,169 @@ class UserController extends Controller {
 }
 ```
 
+## Advanced Usage
+If you have an API that doesn't follow the conventions of this driver you need to configure some more things.
+We have defined some entity annotations which you can use to define target URLs.
+
+```php
+namespace CircleBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Circle\DoctrineRestDriver\Annotations as REST;
+
+/**
+ * This annotation marks the class as managed entity:
+ *
+ * @ORM\Entity
+ * @ORM\Table("products")
+ * @REST\Get("http://www.yourSite.com/api/products/findOne/{id}")
```

Comment:
du implizierst aber, dass ich den kontext restdriver hab - ne route annotation is aber gar nich so selten denke ich und das kann hart verwirren, daher gehts mir hier eher darum semantisch klarzumachen: Das gehört zur ORM und sagt, wie die abfrage gestaltet werden soll. Ich bin am ehesten bei DataSource. Das ding is: auf der semantischen ebene reden wir von orm/sql und nich von rest, daher muss die konifuration der datasource auch mit passendem vokabular stattfinden, aber da find ich nix gutes, idee:

```php
@DataSource\Fetch
@DataSource\Select
@DataSource\Insert
@DataSource\Update
@DataSource\Delete
```